tests/providers/gcp/gcp_test.py

```diff
@@ -634,8 +695,75 @@ def testExecuteQuery(self, mock_gcl_api):
     self.assertEqual(FAKE_LOG_ENTRIES[0], query_logs[0])
 
 
+class GoogleCloudStorageTest(unittest.TestCase):
+  """Test Google Cloud Storage class."""
+  # pylint: disable=line-too-long
+
+  @typing.no_type_check
+  def testSplitGcsPath(self):
+    """Tests that GCS path split is correctly done."""
+    bucket, object_uri = FAKE_GCS.SplitGcsPath(FAKE_GCS_PATH)
+    self.assertEqual(object_uri, FAKE_GCS_URI)
```

Comment:
In this test and the followings, please format equal assertions as `assertEqual(expected, actual)` and use the raw value if known/if possible:

```
self.assertEqual('fake-folder/fake-object', object_uri)
```

This makes it easier to read and know what to expect when reading through the test file :)

Reply:
+1, see comments below too